scripts/ipintutil

```diff
@@ -0,0 +1,265 @@
+#!/usr/bin/env python3
+
+import argparse
+import os
+import subprocess
+import sys
+
+import netaddr
+import netifaces
+from natsort import natsorted
+from tabulate import tabulate
+
+from sonic_py_common import multi_asic
+from swsssdk import ConfigDBConnector, SonicDBConfig
+from utilities_common import constants
+from utilities_common import multi_asic as multi_asic_util
+
+
+try:
+    if os.environ["UTILITIES_UNIT_TESTING"] == "2":
+
+        modules_path = os.path.join(os.path.dirname(__file__), "..")
+        tests_path = os.path.join(modules_path, "tests")
+        sys.path.insert(0, modules_path)
+        sys.path.insert(0, tests_path)
+        import mock_tables.dbconnector
+        if os.environ["UTILITIES_UNIT_TESTING_TOPOLOGY"] == "multi_asic":
+            import mock_tables.mock_multi_asic
+            mock_tables.dbconnector.load_namespace_config()
+except KeyError:
+    pass
+
+
+def get_bgp_peer():
+    """
+    collects local and bgp neighbor ip along with device name in below format
+    {
+     'local_addr1':['neighbor_device1_name', 'neighbor_device1_ip'],
+     'local_addr2':['neighbor_device2_name', 'neighbor_device2_ip']
+     }
+    """
+    bgp_peer = {}
+    config_db = ConfigDBConnector()
+    config_db.connect()
+    data = config_db.get_table('BGP_NEIGHBOR')
+
+    for neighbor_ip in data.keys():
+        local_addr = data[neighbor_ip]['local_addr']
+        neighbor_name = data[neighbor_ip]['name']
+        bgp_peer.setdefault(local_addr, [neighbor_name, neighbor_ip])
+    return bgp_peer
+
+def cmd_exec_in_root():
+    if os.geteuid() != 0 and os.environ.get("UTILITIES_UNIT_TESTING", "0") != "2":
+        sys.exit("Root privileges required for this operation")
+
+def skip_ip_intf_display(interface, display_option):
+
+    if display_option != constants.DISPLAY_ALL:
+        if interface.startswith('Ethernet') and multi_asic.is_port_internal(interface):
+            return True
+        elif interface.startswith('PortChannel') and multi_asic.is_port_channel_internal(interface):
+            return True
+        elif interface.startswith('Loopback4096'):
+            return True
+        elif interface.startswith('eth0'):
+            return True
+        elif interface.startswith('veth'):
+            return True
+    return False
+
+
+def get_if_admin_state(iface, namespace):
+    """
+    Given an interface name, return its admin state reported by the kernel
+    """
+    cmd = "cat /sys/class/net/{0}/flags".format(iface)
+    if namespace != constants.DEFAULT_NAMESPACE:
+        cmd = "sudo ip netns exec {} {}".format(namespace, cmd)
+    try:
+        proc = subprocess.Popen(
+            cmd, shell=True, stderr=subprocess.STDOUT, stdout=subprocess.PIPE)
```

Comment:
Without `text=True` parameter, stdout/stderr will be returned as bytes, not strings in Python 3. Is this intended?